packages/zoe/src/zoe.js

```diff
@@ -212,14 +223,23 @@ import { makeTables } from './state';
  * queries based on other information, we choose to omit it. For
  * instance, `installationHandle` can be derived from
  * `instanceHandle` and is omitted even though it is useful.
- * @param  {object} seat - an object defined by the smart
+ * @param  {OfferHook} offerHook - a function that will be handed the seat at
+ * the right time, and returns a contract-specific value that will be put in the OfferResultRecord.
+ * The seat is an object defined by the smart
  * contract that is the use right associated with the invite. In
  * other words, buying the invite is buying the right to call
  * methods on this object.
```

Comment:
There are no seats.

Delete lines 228-231

Reply:
addressed in 43aa15b1